pontoon/settings/base.py

```diff
@@ -487,9 +486,45 @@ def _allowed_hosts():
 # Always generate a CSRF token for anonymous users.
 ANON_ALWAYS = True
 
+# Set X-Frame-Options to DENY by default on all responses.
+X_FRAME_OPTIONS = 'DENY'
+
 # Use correct header for detecting HTTPS on Heroku.
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
+# Strict-Transport-Security: max-age=63072000
+# Ensures users only visit the site over HTTPS
+SECURE_HSTS_SECONDS = 63072000
+
+# X-Content-Type-Options: nosniff
+# Disables browser MIME type sniffing
+SECURE_CONTENT_TYPE_NOSNIFF = True
+
+# x-xss-protection: 1; mode=block
+# Activates the browser's XSS filtering and helps prevent XSS attacks
+SECURE_BROWSER_XSS_FILTER = True
+
+# Content-Security-Policy headers
+CSP_DEFAULT_SRC = ("'none'",)
+CSP_CHILD_SRC = ("https:",)
+CSP_FRAME_SRC = ("https:",)  # Older browsers
+CSP_CONNECT_SRC = ("'self'",)
+CSP_FONT_SRC = ("'self'",)
+CSP_IMG_SRC = (
+    "'self'",
+    "https://*.wp.com/pontoon.mozilla.org/",
```

Comment:
It seems to be largely the same as this image, fwiw: https://pontoon.mozilla.org/static/img/anon.0a768246b7e0.jpg